tests/manifests/golden-pvc.yml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: cdi-test
+objects:
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: ${PVC_NAME}
+    annotations:
+      kubevirt.io/storage.import.endpoint: "${EP_URL}"
+      kubevirt.io/storage.import.secretName: "${EP_SECRET}"
```

Comment:
Are these annotations correct for the current version of CDI that the CI is using?

Reply:
The current "golden" pvc endpoint annotation is `cdi.kubevirt.io/storage.import.endpoint` (as shown in the cdi README file).
```

```

Reply:
okay, just curious why seems the old annotation also works? but anyway I changed to the new one.